CondTools/SiStrip/plugins/SiStripPedestalsReader.cc

```diff
@@ -1,4 +1,32 @@
-#include "CondTools/SiStrip/plugins/SiStripPedestalsReader.h"
+// system include files
+//#include <memory>
+#include <iostream>
+#include <cstdio>
+#include <sys/time.h>
+
+// user include files
+#include "CondFormats/DataRecord/interface/SiStripPedestalsRcd.h"
+#include "CondFormats/SiStripObjects/interface/SiStripPedestals.h"
+#include "FWCore/Framework/interface/ESHandle.h"
+#include "FWCore/Framework/interface/Event.h"
+#include "FWCore/Framework/interface/EventSetup.h"
+#include "FWCore/Framework/interface/Frameworkfwd.h"
+#include "FWCore/Framework/interface/one/EDAnalyzer.h"
+#include "FWCore/MessageLogger/interface/MessageLogger.h"
+#include "FWCore/ParameterSet/interface/ParameterSet.h"
+#include "FWCore/ServiceRegistry/interface/Service.h"
+
+class SiStripPedestalsReader : public edm::one::EDAnalyzer<> {
+public:
+  explicit SiStripPedestalsReader(const edm::ParameterSet&);
+  ~SiStripPedestalsReader() override = default;
+
+  void analyze(const edm::Event&, const edm::EventSetup&) override;
+
+private:
+  uint32_t printdebug_;
```

Comment:
shouldn't this be a bool after all?

Reply:
No, as you can see here it's used to limit the amount of printout.

https://github.com/cms-sw/cmssw/blob/8583b733fd4047cdd9fbeb285110bdae83c73c07/CondTools/SiStrip/plugins/SiStripPedestalsReader.cc#L19

I am dismissing your comment.